www/routes/raw.ts

```diff
@@ -37,23 +38,20 @@ export const handler: Handlers = {
       return response;
     }
 
-    if (isHTML) {
-      response.headers.set("Content-Type", "text/plain");
-    } else if (path.endsWith(".ts")) {
-      response.headers.set("Content-Type", "application/typescript");
-    } else if (path.endsWith(".js")) {
-      response.headers.set("Content-Type", "application/javascript");
-    } else if (path.endsWith(".tsx")) {
-      response.headers.set("Content-Type", "text/tsx");
-    } else if (path.endsWith(".jsx")) {
-      response.headers.set("Content-Type", "text/jsx");
-    } else if (path.endsWith(".json")) {
-      response.headers.set("Content-Type", "application/json");
-    } else if (path.endsWith(".wasm")) {
-      response.headers.set("Content-Type", "application/wasm");
-    } else {
-      response.headers.set("Content-Type", "text/plain");
+    const contentTypes = new Map([
+      [".html", "text/plain"],
+      [".ts", "application/typescript"],
+      [".js", "application/javascript"],
+      [".tsx", "text/tsx"],
+      [".jsx", "text/jsx"],
+      [".json", "application/json"],
+      [".wasm", "application/wasm"],
+    ]);
+    if(isHTML){
```

Comment:
This doesn't look correct. We should always check if we have a content type value for a given extension like we did before, not only when we're dealing with HTML files.